packages/client/src/components/signin-dialog.vue

```diff
@@ -2,7 +2,7 @@
 <XModalWindow ref="dialog"
 	:width="370"
 	:height="400"
-	@close="dialog.close()"
+	@close="onClose"
 	@closed="emit('closed')"
```

Comment:
Couldn't you have use the existing `closed` event?

Reply:
From the PR description:

> As mentioned in my issue, I adjusted `XSigninDialog` to also emit the `close` event, which is required to handle what happens on manual closing of the dialog.

`closed` always gets emitted when the dialog is closed, which also happens after a successful login, so this would trigger a redirect after successfully logging in, and from my tests that caused race conditions with actually setting the account data (or: the login dialog would always show up)

`close` is a different event triggered when closing the modal manually, which fits the use case here.

Reply:
Okay, but then I still think the event is named confusingly. Maybe `cancelled` is a better name?

Reply:
I agree. Should I rename this just local to the sign-in modal for now? I wouldn't want to go further than that because as soon as we get to `XModalWindow` there's quite a few Vue handlers on that event (I was able to easily search for `@close=`) and I'd rather refactor that in it's own PR.

Reply:
Yes, I was only referring to the new event you created.

Reply:
Renamed the event that is emitted to `cancelled` now.

Leaving the handler as `onClose` however, as that is the parent event name still.